spec/requests/api/v3/root_resource_spec.rb

```diff
@@ -65,11 +74,22 @@
       end
 
       it 'should respond with 200' do
-        expect(subject.status).to eq(200)
+        expect(response.status).to eq(200)
       end
 
       it 'should respond with a root representer' do
-        expect(subject.body).to have_json_path('instanceName')
+        expect(subject).to have_json_path('instanceName')
+      end
+
+      it_behaves_like 'has a titled link' do
+        let(:link) { 'user' }
+        let(:href) { api_v3_paths.user(current_user.id) }
+        let(:title) { current_user.name }
+      end
+
+      it_behaves_like 'has an untitled link' do
+        let(:link) { 'userPreferences' }
+        let(:href) { api_v3_paths.my_preferences }
```

Comment:
Aha, here are those specs I was missing in the representer specs...

For efficiency reasons I would suggest to move those specs to the representer specs, as they are generally faster. (and also the concern under test does belong to the representer, rather than the endpoint)